src/commands/compose.ts

```diff
@@ -61,7 +61,14 @@ async function compose(commands: ('up' | 'down')[], message: string, dockerCompo
         }
 
         const items: vscode.QuickPickItem[] = computeItems(folder, uris);
-        selectedItems = [<Item>await ext.ui.showQuickPick(items, { placeHolder: `Choose Docker Compose file ${message}` })];
+
+        if ((items.length === 1 && isDefaultDockerComposeFile(items[0].label))
+            || (items.length === 2 && items.some(i => isDefaultDockerComposeFile(i.label)) && items.some(i => isDefaultDockerComposeOverrideFile(i.label)))) {
+            // if the current set of docker files contain only docker-compose.yml or docker-compose.yml with override file,
+            // don't ask user for a docker file and let docker-compose automatically pick these files.
+        } else {
+            selectedItems = [<Item>await ext.ui.showQuickPick(items, { placeHolder: `Choose Docker Compose file ${message}` })];
```

Comment:
From this prompt it is not clear for the user that files will be used in order of selection, but it does matter for Docker Compose. How about

"Select Docker Compose file(s). Files will be used in the order in which they were selected."

It is a bit verbose, but I cannot think of anything better/clearer 🤔

Reply:
The compose command can be executed in 2 ways. Right click on the docker-compose*.yml files and choose the context menu or using command palette.
At this time multi selection is allowed only during context menu and not during command palette execution. The order of files selected are preserved while executing from context menu.

Reply:
Got it, thanks. Please disregard my suggestion then

Reply:
updated the description to make it clear.